tools/autograd/derivatives.yaml

```diff
@@ -404,12 +404,12 @@
   self: grad / other
 
 - name: dot(Tensor self, Tensor tensor) -> Tensor
-  self: grad * tensor
-  tensor: grad * self
+  self: correct_dtype_gradients(self.scalar_type(), grad * tensor.conj())
+  tensor: correct_dtype_gradients(tensor.scalar_type(), grad * self.conj())
 
 - name: vdot(Tensor self, Tensor other) -> Tensor
```

Comment:
What is vdot actually doing? It is not documented anywhere....

Reply:
https://pytorch.org/docs/master/generated/torch.vdot.html#torch.vdot

Reply:
Ho it's only on master! Thanks for the link.